PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -189,7 +189,7 @@ def miniAOD_customizeCommon(process):
 	 valueLabels = cms.vstring('pt','emEnergyFraction'),
 	 lazyParser = cms.bool(True) )
     task.add(process.caloJetMap)
-    process.patJets.userData.userFloats.src += [ cms.InputTag("caloJetMap:pt"), cms.InputTag("caloJetMap:emEnergyFraction") ]
+#    process.patJets.userData.userFloats.src += [ cms.InputTag("caloJetMap:pt"), cms.InputTag("caloJetMap:emEnergyFraction") ]
```

Comment:
I think that it is still needed:
- the threshold in slimmed calojets is not necessarily the same
- it would take an extra effort to loop over the slimmed calojets to get a match and compute these userfloats